src/Main.java

```diff
@@ -4,6 +4,13 @@ public static void main(String[] args) {
 
         for (int i = 0; i <= 100 ; i++) {
             System.out.println("TIC");
+
+            try{
+                Thread.sleep(1000);
+            } catch (InterruptedException e) {
+                Thread.currentThread().interrupt();
+            }
+
```

Comment:
this try catch poluates the code too much.
you should add an exception thrown at the method name